src/client/index.js

```diff
@@ -287,7 +287,15 @@ export const signOut = async (args = {}) => {
   const res = await fetch(`${baseUrl}/signout`, fetchOptions)
   const data = await res.json()
   _sendMessage({ event: 'session', data: { trigger: 'signout' } })
-  window.location = data.url ?? callbackUrl
+  const finalLocation = new URL(data.url ?? callbackUrl)
+  window.location = finalLocation
+  if (
```

Comment:
I'm not sure I understand the need for this if statement. wouldn't setting window.location trigger a reload?

Reply:
It won't trigger if the url is the same page and contains `#`.
Example:
| Current  | Final | window.location trigger reload |
| ------------- | ------------- | ------------- |
| /  | /  | yes |
| / | /#  | no |
| /# | /#  | no |
| / | /other  | yes |
| / | /other#  | yes |

Reply:
So I recently came across this at work and remembered your PR! Do you know if this is the intended browser behavior, or some kind of bug?

I can verify it very easily but opening the inspector on any page that has a hashtag in the url, it won't reload!